src/analytics/analytics.ts

```diff
@@ -65,10 +65,11 @@ export class AngularFireAnalytics {
 
     if (!analyticsInitialized) {
       if (isPlatformBrowser(platformId)) {
-        gtag = (window[GTAG_FUNCTION_NAME] as any) || ((...args: any[]) => {
-          (window[DATA_LAYER_NAME] as any).push(args);
-        });
         window[DATA_LAYER_NAME] = window[DATA_LAYER_NAME] || [];
+        // tslint:disable-next-line: only-arrow-functions
+        gtag = (window[GTAG_FUNCTION_NAME] as any) || (function(..._args: any[]) {
+          (window[DATA_LAYER_NAME] as any).push(arguments);
+        });
```

Comment:
We really should have a comment in here that warns developers/contributors to making this an arrow function causes GA to break completely.

In the current state, it's likely that someone will come in and "fix" this breaking GA again.

Reply:
You are right, I can make and submit another PR when I have free time